monitor/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '2.4'
+
+networks:
+  # This networks defined in main yaml docker-compose.yml file
+  # Since both are refering common project the network defined here will refer the same.
+  default:
+    ipam:
+      driver: default
+
+# YAML aliases
+x-default: &default
+  dns:
+    - 1.1.1.1
+    - 1.0.0.1
+
+services:
+
+  monitor:
+    <<: *default
```

Comment:
Add `build: .` to build from local. 

To run this in server. this yaml has to be there in compose-deployment repo with the name `backup-monitor.yml` . Add the readme to start the monitor as like we did for backup.

Reply:
created a PR for that too https://github.com/tamil-wiki/compose-deployment/pull/97